GoogleSignIn/Tests/Unit/OIDTokenResponse+Testing.h

```diff
@@ -53,6 +53,8 @@ extern NSString * const kFatPictureURL;
 
 + (instancetype)testInstanceWithIDToken:(NSString *)idToken;
 
++ (instancetype)testInstanceWithAccessTokenExpiresIn:(NSNumber *)accessTokenExpiresIn;
```

Comment:
For your consideration: `testInstanceWithAccessTokenExpiration:`.

Also, should the parameter's type be `NSNumber`? I think you're passing in a `NSTimeInterval`?

Reply:
Also, you can rename the parameter to `expiration`.

Reply:
I think it's okay as `NSNumber` since I'm passing in the variable using the `@()` literal on the `NSTimeInterval` making it the expected type. I followed the way `GIDGoogleUserTest` went about passing in the expiration [here](https://github.com/google/GoogleSignIn-iOS/blob/b0a724e1a8e12cfbe027d8f844a1e6f80f17532f/GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m#L190-L193) to keep it consistent. Let me know if I should change.

Reply:
I see. It looks like the implementation calls through to [`+[OIDTokenResponse testInstanceWithIDToken:accessToken:expiresIn:refreshToken:tokenRequest:]`](https://github.com/google/GoogleSignIn-iOS/blob/main/GoogleSignIn/Tests/Unit/OIDTokenResponse%2BTesting.m#L68), which expects an `NSNumber`. That's because it gets put into a dictionary.